server/ansible/roles/pbench-server-vars/tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_logs_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_logs_dir }}"
+    verbosity: 1
+
+# pbench tmp dir
+- name: fetch the tmp directory
+  shell: getconf.py pbench-tmp-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_tmp_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_tmp_dir }}"
+    verbosity: 1
+
+# pbench quarantine dir
+- name: fetch the quarantine directory
+  shell: getconf.py pbench-quarantine-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_quarantine_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_quarantine_dir }}"
+    verbosity: 1
+
+# pbench hostinfo prefix
+- name: fetch the host info prefix
+  shell: getconf.py host-info-path-prefix results
+  register: cmd_output
+  ignore_errors: yes
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_host_info_prefix: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_host_info_prefix }}"
+    verbosity: 1
+
+# pbench reception area
+- name: fetch the reception area directory
+  shell: getconf.py pbench-receive-dir-prefix pbench-server
+  register: cmd_output
+  ignore_errors: yes
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_reception_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_reception_dir }}"
+    verbosity: 1
+
+# httpd document root
+- name: fetch the httpd document root directory
+  shell: getconf.py documentroot apache
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    httpd_document_root_dir: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+
+- debug:
+    msg: "{{ httpd_document_root_dir }}"
+    verbosity: 1
+
+
+# pbench owner
+- name: fetch the pbench owner
+  shell: getconf.py user pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_user: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_user }}"
+    verbosity: 1
+
+# pbench group
+- name: fetch the pbench group
+  shell: getconf.py group pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
+
+- set_fact:
+    pbench_group: "{{ cmd_output.stdout_lines[0].strip() }}"
+
+- debug:
+    msg: "{{ pbench_group }}"
+    verbosity: 1
+
+# pbench host
+- name: fetch the pbench server
+  shell: getconf.py host pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
```

Comment:
Is the `/opt/pbench-server` configurable?

Reply:
See above.